client/src/pages/trainee/dashboard/index.tsx

```diff
@@ -0,0 +1,42 @@
+import Tabs from '@/src/shared/components/Tabs';
+import Tab from '@/src/shared/components/Tabs/Tab';
+import Link from 'next/link';
+import { Fragment } from 'react';
+
+const DashboardPage: React.FC = () => {
+  return (
+    <Fragment>
+      <div className="flex flex-col justify-center items-center gap-24 w-full pt-24">
+        <div className="flex flex-col gap-2 justify-center items-center">
+          <div className="font-semibold text-[32px] leading-[40px] font-lora">
+            <span className="text-dark">Welcome back, </span>
+            <span className="text-red">Trainee Franz</span>
```

Comment:
There is already user name in next auth session, so please make the user name dynamic